web-server/src/controllers/tasks/create.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ id: number;
+  title: string;
+  slug: string;
+  description: string;
+  statement: string;
+  */
+import { Request, Response, NextFunction } from 'express';
+
+import { AppDataSource } from 'orm/data-source';
+import { Task } from 'orm/entities/tasks/Task';
+import { CustomError } from 'utils/response/custom-error/CustomError';
+
+export const create = async (req: Request, res: Response, next: NextFunction) => {
+  const { ownerId, title, slug, description, statement } = req.body;
```

Comment:
You should not trust that the client sends legitimate data. There could be evil hackers in our midst who will send bogus `ownerId`s. For a programming website, this is 100x more common.

Reply:
You should instead get the ownerId from the request current user which should be accessible with `req.jwtPayload`